### PR TITLE
Faq section

### LINKED ISSUE #13: FAQ Section

### PR DESCRIPTION
**Pull request by mlodyoskar**
https://user-images.githubusercontent.com/45982530/192982862-983d6b19-4cfd-4039-af43-f5c5fb3c27a2.mp4

Closes #13

### CODE REVIEW

File: src/components/main-page/FAQ/FAQDisclosure/FAQDisclosure.module.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+.panel {
+  background-color: var(--color-white);
+  color: var(--color-black);
+  padding: var(--space-m);
+  border-radius: 0 0 var(--border-radius-default) var(--border-radius-default);
+  animation-delay: 0.2s;
+}
+
+.enter {
+  transition: all 0.2s ease-out;
```
Comment (roertbb) [inline src/components/main-page/FAQ/FAQDisclosure/FAQDisclosure.module.scss:43]: I'm wondering if it wouldn't look better if we'd increase the values a little bit - we could check out if with the values from `Navigation` it wouldn't look better, wdyt?

File: src/components/main-page/FAQ/FAQ.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  question: string;
+  answer: string;
+};
+
+interface Section {
+  title: string;
+  questions: Question[];
+}
+
+const FAQ = () => {
```
Comment (roertbb) [inline src/components/main-page/FAQ/FAQ.tsx:16]: similarly as in comment below
```suggestion
export const FAQ = () => {
```

File: src/components/main-page/FAQ/FAQ.module.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.heading {
+  margin-bottom: var(--space-2xl);
+}
+
+.container {
```
[the remark this answers is not in the document]
Comment (mlodyoskar) [reply src/components/main-page/FAQ/FAQ.module.scss:5]: Sure, probably I've changed my mind during developing and forgot to delete this class, thanks for spotting it!

File: src/components/ui/Navigation/Navigation.module.scss [hunk 4]
```diff
@@ -3,7 +3,7 @@
   top: 0;
   width: 100%;
   height: var(--navbar-height);
-  z-index: 1;
+  z-index: 100;
```
Comment (mlodyoskar) [inline src/components/ui/Navigation/Navigation.module.scss:6]: Without it, the faq disclosure text was above navigation

**Review by roertbb — APPROVED** [review]
It's highly recommended to add it to the `work-in-progress` route 😉

**Pull request merged**